### ISSUE TITLE
Error in CellCorHeatmap()

### ISSUE THREAD

**Issue by redtorrentCN** [zhanghao-njmu/SCP#231]
Hi, sir,
When  I run CellCorHeatmap(), a trouble occurred like following:

> Use 'raw' method to find neighbors.
Predict cell type...
Error in if (length(x = value) == ncol(x = x) && all(names(x = value) ==  : 
  missing value where TRUE/FALSE needed`

this is my code:

> subnatmed.epi <- subset(natmed,celltype %in% c('Uncili_Epi','Cili_Epi'))
DimPlot(subnatmed.epi,reduction = 'umap.scvi',label = T,pt.size = 1.5)
subnatmed.epi$Epitype <- ifelse(subnatmed.epi$scvi_clusters_0.1 %in% 4,'Cili_Epi',
                                ifelse(subnatmed.epi$scvi_clusters_0.1 %in% 1,'Uncili_Epi 1#','Uncili_Epi 2#'))

> subAsse.epi <- subset(Asse,celltype %in% c('Uncili_Epi','Cili_Epi'))
DimPlot(subAsse.epi,reduction = 'umap.scvi',label = T,pt.size = 1.5)
subAsse.epi$Epitype <- ifelse(subAsse.epi$scvi_clusters_1 %in% 21,'Cili_Epi 1#',
                                ifelse(subAsse.epi$scvi_clusters_1 %in% 22,'Cili_Epi 2#',
                                       ifelse(subAsse.epi$scvi_clusters_1 %in% 8,'Uncili_Epi 1#',
                                              ifelse(subAsse.epi$scvi_clusters_1 %in% 14,'Uncili_Epi 2#','Uncili_Epi 3#'))))

> ht <- CellCorHeatmap(
  srt_query =subAsse.epi, srt_ref = subnatmed.epi,
  query_group = "Epitype", ref_group = "Epitype",
  nlabel = 3, label_by = "row",
  show_row_names = TRUE, show_column_names = TRUE
)

I just use seurat  object (s3 assay) directly to SCP package. 
thanks for your reading and  answers. @zhanghao-njmu

**Comment by ziyuan-ma** [comment]
I’m encountering the same issue when trying to run the `CellCorHeatmap()` function. Have you found any potential solutions or workarounds for this problem? Any help would be appreciated. Thanks!
![Xnip2024-10-16_12-01-44](https://github.com/user-attachments/assets/2ed86b51-de06-4b2b-9898-c1cf07232ede)

**Comment by redtorrentCN** [comment]
> I’m encountering the same issue when trying to run the `CellCorHeatmap()` function. Have you found any potential solutions or workarounds for this problem? Any help would be appreciated. Thanks! ![Xnip2024-10-16_12-01-44](https://private-user-images.githubusercontent.com/98864213/377136436-2ed86b51-de06-4b2b-9898-c1cf07232ede.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0ODk1NTYsIm5iZiI6MTcyOTQ4OTI1NiwicGF0aCI6Ii85ODg2NDIxMy8zNzcxMzY0MzYtMmVkODZiNTEtZGUwNi00YjJiLTk4OTgtYzFjZjA3MjMyZWRlLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIxVDA1NDA1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjZGY3NzUxNTcxYjMyNDdjOWYyN2VmMDA5Y2U3ODRlZDZhNWVlZDgwMzk4M2Q1NGZhZTBmNjY5ZjAxNzRjY2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.OcQvkRftMJtyuAni5t7mdwwQdh9AqQ6L7kPx_RpFVHg)

Yes, just downgrade version to 0.2.6. 'CellCorHeatmap()` will be fine.
